package.json

```diff
@@ -47,7 +47,7 @@
     "@types/js-beautify": "^1.8.0",
     "@types/lodash": "^4.14.119",
     "@types/mustache": "^0.8.32",
-    "@types/node": "^10.12.18",
+    "@types/node": "^12.12.6",
```

Comment:
Why bump this to 12? Do you need to use 12 here? I think we are still on 10 for our official local builds...

Reply:
I was running into this issue and it sounded like the solution was just to bump `@types/node`, https://github.com/microsoft/TypeScript/issues/32333 🤷. I bumped a few other packages over the course of this PR though, so lemme see if it's still an issue

Reply:
Yep, no longer necessary. Will bump this back down